src/test/java/bio/terra/MediaTypeUtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package bio.terra;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.fail;
+
+import bio.terra.cloudfunctions.utils.MediaTypeUtils;
+import bio.terra.common.BaseTest;
+import java.io.BufferedInputStream;
+import java.util.logging.Logger;
+import org.apache.commons.compress.archivers.ArchiveEntry;
+import org.apache.commons.compress.archivers.ArchiveException;
+import org.apache.commons.compress.archivers.ArchiveInputStream;
+import org.apache.commons.compress.compressors.CompressorInputStream;
+import org.junit.Test;
+
+public class MediaTypeUtilsTest extends BaseTest {
```

Comment:
I like that we're testing code around processing a gzip file, but we don't actually test the code that will be deployed to production to process the gzip files.  Let's write some test code that will invoke the code used in `TestRunnerStreamingApp.process()`  Let me know if you want some help doing that.  One approach would be to mock `BigQueryUtils.streamToBQ()` and verify that it is invoked with the arguments expected in the order expected.

Reply:
I'll hold off on this for now to think through what we actually need for unit testing the `process()` method, now `processMessage()` after latest updates.